app/controllers/api/v1/sessions_controller.rb

```diff
@@ -29,18 +32,14 @@ def create
       # DELETE /api/v1/sessions/signout
       def destroy
         session[:user_id] = nil
+        cookies.encrypted[:_extended_session] = nil
```

Comment:
Same here you can create `sign_out` private method to clean a bit the action and be consistent with the `sign_in`.
Besides, setting the `cookies.encrypted[:_extended_session] = nil` will not remove the cookie you can use the `cookies.delete(:_COOKIE_KEY)` API for that.

Reply:
Something like:
```
      # Signs Out the user
      def sign_out
        reset_session
        cookies.delete(:_extended_session)
      end
````